h2o-assemblies/main/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     compile project(":h2o-persist-s3")
     compile project(":h2o-persist-http")
     compile project(":h2o-persist-hdfs")
+    compile project(":h2o-hive")
```

Comment:
Do we want the support for hive in standalone as well? What does it do to dependencies? Are we going to be compatible with any version of Hive?

I would rather start just with Hadoop support.

Reply:
good idea, will move it